sort/Program.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+
+namespace sort
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int i;
+            int[] a = new int[30];
+            Console.Write("Enter the Number of values to be Sort : ");
+            // read the string value and convert it in to integer  
+            int n = Convert.ToInt16(Console.ReadLine());
+            //Reading the values one by one  
+            for (i = 1; i <= n; i++)
```

Comment:
Remember, everything in C# is zero based indexed. So you are actually creating an array with an unused slot.